### PR TITLE
API : Stake | get stake instantaneous rewards

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
image

File: src/test/java/tests/StakeKeyTests.java [hunk 1]
```diff
@@ -31,4 +33,45 @@ public Object[][] DatasetListAddress(){
...
     }
+    @Test(description = "get stake instantaneous rewards with stake key", groups = {"stakeKey"}, dataProvider = "stakeKeyInstantaneousReward")
+    public void getStakeInstantaneousReward(Object stakeKey){
+        stakeKeySteps.getStakeInstantaneousRewards(stakeKey)
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+    }
+    @DataProvider(name = "stakeKeyInstantaneousReward")
+    public Object[][] DatasetStakeKeyInstantaneousRewards(){
+        return new Object[][]{
+                {"stake_test1urz3dwcwdhvl9dy75rj5krvpmjyldeyxqx5p6xms6xyjd7quqmq2p"},
```
Comment (Sotatek-GiangNguyen7) [inline src/test/java/tests/StakeKeyTests.java]: should put on top

File: src/test/java/tests/StakeKeyTests.java [hunk 2]
```diff
@@ -31,4 +33,45 @@ public Object[][] DatasetListAddress(){
...
+                {"12345"}
+        };
+    }
+    @Test(description = "get stake instantaneous rewards with param", groups = "stakeKey", dataProvider = "paramInstantaneousReward")
+    public void getStakkeInstantaneousRewardParam(Object page, Object size){
+        Map<String, Object> param = new CreateParameters()
+                .withPage(page)
+                .withPageSize(size)
+                .getParamsMap();
+        String stakeKey = "stake_test1urz3dwcwdhvl9dy75rj5krvpmjyldeyxqx5p6xms6xyjd7quqmq2p";
```
Comment (Sotatek-GiangNguyen7) [inline src/test/java/tests/StakeKeyTests.java]: should put on top

**Review by Sotatek-GiangNguyen7 — APPROVED** [review]

File: src/test/java/tests/StakeKeyTests.java (hunk 1 above)
Comment (Sotatek-anhnguyen37) [reply src/test/java/tests/StakeKeyTests.java]: this stakeKey is a invalid, so i think we shouldn't put on top

**Comment by ghost** [comment]
image

**Review by Sotatek-GiangNguyen7 — APPROVED** [review]

**Pull request merged**